test/Microsoft.AspNet.Mvc.IntegrationTests/BinderTypeBasedModelBinderIntegrationTest.cs

```diff
@@ -12,89 +12,11 @@ namespace Microsoft.AspNet.Mvc.IntegrationTests
 {
     public class BinderTypeBasedModelBinderIntegrationTest
     {
-        // The NullModelBinder and NullModelNotSetModelBinder return a non null ModelBindingResult but a null model.
-        [Theory(Skip = "ModelBindingResult should be non null if a model binder returns a non null resul #2473.")]
-        [InlineData(typeof(NullModelBinder), true)]
-        [InlineData(typeof(NullModelNotSetModelBinder), false)]
-        public async Task BindParameter_WithModelBinderType_NoData(Type modelBinderType, bool isModelSet)
```

Comment:
Mostly irrelevant since the `CompositeModelBinder` should return `null` result in this case. I could change the assertions if we care about the scenario. Looked unimportant to me but I'm fine either way.